src/index.js

```diff
@@ -14,41 +14,54 @@ function layer(context, selectedLayer) {
   };
 }
 
-function styleguideColors(context, colors) {
+function textStyles(context) {
+  var code = "// MARK: - Project Texts styles \n\n";
+  
+  var textStylesProject = context.project.textStyles
+  var textStylesAll = textStylesProject.concat(context.project.linkedStyleguide.textStyles)
 
-  var code = "// MARK: - Project Color palette\n\n";
-  code += swiftColors.getColorsSwiftSnippet(context, colors, false);
+  code += swiftTextStyles.getTextStylesSwiftSnippet(context, textStylesAll, false)
 
   return {
     code: code,
     language: "swift"
   };
 }
 
-function styleguideTextStyles(context, textStyles) {
-  var code = "// MARK: - Project Texts styles\n\n";
-  code += swiftTextStyles.getTextStylesSwiftSnippet(context,textStyles, false)
+function exportTextStyles(context) {
+  var textStylesProject = context.project.textStyles
+  var textStylesAll = textStylesProject.concat(context.project.linkedStyleguide.textStyles)
```

Comment:
From the documentation I see 3 different ways to get textStyles:
- context.project.textStyles
- context.styleguide.textStyles
- context.project.linkedStyleguide.textStyles
You seem to use only 2 of them, any reason?

Reply:
The second case is not working. Only from `project` and `project.linkedStyleguide`. But I'll check it. Thanks!